src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -232,13 +233,23 @@ export class MeetingManager implements AudioVideoObserver {
     this.meetingRegion = meetingInfo.MediaRegion;
     this.meetingId = this.configuration.meetingId;
 
-    if (eventReporter) {
-      this.eventReporter = eventReporter;
+    if (eventController) {
+      if(this._eventController)
+      {
+        this._eventController.removeObserver(this.eventDidReceiveRef);
```

Comment:
I think this should be enough and only approach for now. As discussed JS SDK needs to get this fixed.